[devops] Add deployment files and info (#25)
rsksmart/rif-relay-server: deploy/README.md

# Deployment information

> * **Note**: Information for deploying the `rif-relay-server` and a `relaying-services-sdk-dapp` to a server, not meant to be used locally.

Reviewer: ```suggestion
* **Note**: This information is for deploying `rif-relay-server` and `relaying-services-sdk-dapp` to a server, and not meant to be used locally.
```

* Server must have all the files referenced here and 

> should have an RSK node installed and running locally. 

Author: Generally yes, but I suppose it depends on the node load. It's not strictly required either that each server has its own rskj node or that there is a single node for all the servers.
The important thing is that we cannot use the public RSKj node.

> * `Dockerfile` and `docker-compose.yml` files should be on the root of the repository.

Reviewer: ```suggestion
* `Dockerfile` and `docker-compose.yml` files should be in the repository's root directory.
```
